languages/java/exercises/concept/conditionals/src/main/java/Blackjack.java

```diff
@@ -0,0 +1,32 @@
+public class Blackjack{
+
+    public int  ParseCard(String card) {
```

Comment:
functions names in java normally hasn't capital letter, so prefer `parseCard` instead.
please fix all method names